app/presenters/manual_section_presenter.rb

```diff
@@ -0,0 +1,59 @@
+class ManualSectionPresenter < ContentItemPresenter
+  include ContentItem::Metadata
+  include ContentItem::Manual
+  include ContentItem::ManualSection
+
+  def base_path
+    manual["base_path"]
+  end
+
+  def intro
+    return nil unless details["body"]
+
+    intro = Nokogiri::HTML::DocumentFragment.parse(details["body"])
+
+    # Strip all content following and including h2
+    intro.css("h2").xpath("following-sibling::*").remove
+    intro.css("h2").remove
+
+    intro.text.squeeze == "\n" ? "" : intro
```

Comment:
Is it possible for an intro to have more than one new line at this point i.e could we end up with `"\n \n"`?
Would it be possible to write a test for this new functionality?